Casks/mycard.rb

```diff
@@ -0,0 +1,25 @@
+cask "mycard" do
+  version "3.0.67"
+  sha256 "871eb016dbd6e8b6b6e9d4b6a74218f54f6acc344f5ceee60f9a344df6d9de08"
+
+  url "https://cdn02.moecube.com:444/downloads/MyCard-#{version}.dmg",
+      verified: "cdn02.moecube.com:444/downloads/"
+  name "MyCard"
+  desc "Yu-Gi-Oh! Complete Card Simulator YGOpro · Touhou Project"
```

Comment:
I think the below should suffice.

```suggestion
  desc "Yu-Gi-Oh! Complete Card Simulator"
```